alloydb-jdbc-connector/src/main/java/com/google/cloud/alloydb/DefaultConnectionInfoRepository.java

```diff
@@ -84,6 +85,11 @@ public ConnectionInfo getConnectionInfo(InstanceName instanceName, KeyPair keyPa
         info.getIpAddress(), info.getInstanceUid(), clientCertificate, certificateChain);
   }
 
+  @Override
+  public void close() {
```

Comment:
Does this need to be public?

Reply:
It does. If we remove `public` the error below is displayed: 

`Cannot reduce the visibility of the inherited method from CloseableJava(67109273)`